python/lsst/ip/isr/isrTaskLSST.py

```diff
@@ -1061,32 +1061,21 @@ def getBrighterFatterKernel(self, detector, bfKernel):
             else:
                 raise RuntimeError("Failed to extract kernel from new-style BF kernel.")
         elif bfKernel.level == 'AMP':
-            self.log.warning("Making DETECTOR level kernel from AMP based brighter "
-                             "fatter kernels.")
+            self.log.info("Making DETECTOR level kernel from AMP based brighter "
+                          "fatter kernels.")
             bfKernel.makeDetectorKernelFromAmpwiseKernels(detName)
             bfKernelOut = bfKernel.detKernels[detName]
             return bfKernelOut, bfGains
 
     def applyBrighterFatterCorrection(self, ccdExposure, flat, dark, bfKernel, bfGains):
-        # We need to apply flats and darks before we can interpolate, and
-        # we need to interpolate before we do B-F, but we do B-F without
-        # the flats and darks applied so we can work in units of electrons
-        # or holes. This context manager applies and then removes the darks
-        # and flats.
-        #
-        # We also do not want to interpolate values here, so operate on
-        # temporary images so we can apply only the BF-correction and roll
-        # back the interpolation.
-        # This won't be necessary once the gain normalization
-        # is done appropriately.
+        # We need to interpolate before we do B-F.
```

Comment:
Should probably add a comment that the fwhm parameter is currently unused by the code that is called.